database/migrations/2021_12_09_075855_create_customers_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCustomersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('customers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->char('first_name', 50);
```

Comment:
Why was used char instead of varchar?

Reply:
I thought that char is very similar to varchar, and it doesn't matter which one you use, but then I realized that varchar is more efficient in that case.